frontend/src/features/admin-form/common/components/PreviewFormBanner/PreviewFormBanner.tsx

```diff
@@ -1,39 +1,183 @@
 import { Link as ReactLink } from 'react-router-dom'
-import { Flex, Icon, Text } from '@chakra-ui/react'
+import { Waypoint } from 'react-waypoint'
+import {
+  Divider,
+  Drawer,
+  DrawerBody,
+  DrawerContent,
+  DrawerOverlay,
+  Flex,
+  Icon,
+  IconButton,
+  Portal,
+  Slide,
+  Stack,
+  Text,
+  useDisclosure,
+} from '@chakra-ui/react'
 
-import { FormAuthType } from '~shared/types'
-
-import { BxsEditAlt } from '~assets/icons/BxsEditAlt'
-import { ADMINFORM_ROUTE } from '~constants/routes'
+import { ADMINFORM_ROUTE, DASHBOARD_ROUTE } from '~constants/routes'
+import Button, { ButtonProps } from '~components/Button'
 import Link from '~components/Link'
 
 import { usePublicFormContext } from '~features/public-form/PublicFormContext'
+import { DuplicateFormModal } from '~features/workspace/components/DuplicateFormModal'
 
-export const PreviewFormBanner = (): JSX.Element => {
-  const { form, formId } = usePublicFormContext()
+export const StickyPreviewHeader = ({
+  isOpen,
+}: {
+  isOpen: boolean
+}): JSX.Element => (
+  <Portal>
+    <Slide aria-hidden direction="top" in={isOpen}>
+      <PreviewFormBanner isTemplate />
+    </Slide>
+  </Portal>
+)
+interface PreviewFormBannerProps {
+  isTemplate?: boolean
+}
 
+export const PreviewFormBanner = ({
+  isTemplate,
+}: PreviewFormBannerProps): JSX.Element => {
+  const { formId } = usePublicFormContext()
+  const {
+    isOpen: isModalOpen,
+    onOpen: onModalOpen,
+    onClose: onModalClose,
+  } = useDisclosure()
+  const {
+    isOpen: isDrawerOpen,
+    onOpen: onDrawerOpen,
+    onClose: onDrawerClose,
+  } = useDisclosure()
+  const mobileDrawerButtonProps: Partial<ButtonProps> = useMemo(
+    () => ({
+      isFullWidth: true,
+      iconSpacing: '1rem',
+      justifyContent: 'flex-start',
+      variant: 'clear',
+      colorScheme: 'secondary',
+      textStyle: 'body-1',
+    }),
+    [],
+  )
   return (
-    <Flex bg="success.600" px="2rem" display="flex" width="100%">
-      <Flex align="center" flex={1} justify="space-between" flexDir="row">
-        <Text textStyle="caption-1">
-          Preview{' '}
-          {form?.authType && form.authType !== FormAuthType.NIL
-            ? 'with test data'
-            : 'mode'}
-        </Text>
-        <Link
-          variant="standalone"
-          color="secondary.700"
-          aria-label="Click to edit the form"
-          as={ReactLink}
-          to={`${ADMINFORM_ROUTE}/${formId}`}
-        >
+    <>
+      <Flex bg="primary.100" py="1rem" px="2rem" display="flex" width="100%">
+        <Flex align="center" flex={1} justify="space-between" flexDir="row">
           <Flex align="center">
-            <Icon aria-hidden as={BxsEditAlt} fontSize="1rem" mr="0.75rem" />
-            <Text textStyle="subhead-2">Edit form</Text>
+            <Icon aria-hidden as={BiShow} fontSize="1rem" mr="0.75rem" />
+            <Text textStyle="subhead-3">Template Preview</Text>
```

Comment:
Based on figma, templates should say "Template preview" but form preview should say "Form preview", hahah

Reply:
[d147cf9](https://github.com/opengovsg/FormSG/pull/5377/commits/d147cf94f10e80bfad26efb58c95c08305369a6a)